packages/studioCMS/src/components/exports/navigation.css

```diff
@@ -87,6 +87,33 @@
     }
 }
 
+@media (max-width: 400px) {
+    .navigation {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: auto;
+    }
+    .navigation .title {
+        display: flex;
+        position: relative;
```

Comment:
are these all `position: relative` for a reason??

Reply:
![image](https://github.com/astrolicious/studiocms/assets/30383579/6655d4ae-5ab5-40d8-a24b-cd4a724fb091)
indeed.  this front navigation was having a flow issue where all the elements would stack on smaller screens.... this fixes that for smaller screen sizes...  by unstacking all the position absolute elements